### ISSUE TITLE
langchain_openai not running in Flutter Web: "The integer literal 9223372036854775807 can't be represented exactly in JavaScript"

### ISSUE THREAD

**Issue by bryantwilliam** [davidmigloz/langchain_dart#207]
### System Info

I'm using Flutter Web and I get an error when using langchain_open.dart:

```
/C:/Users/William/AppData/Local/Pub/Cache/hosted/pub.dev/openai_dart-0.0.2+1/lib/src/generated/schema/create_completion_request.dart:147:31: Error: The integer literal 9223372036854775807 can't be represented exactly in JavaScript.
Try changing the literal to something that can be represented in JavaScript. In JavaScript 9223372036854775808 is the nearest value that can be represented exactly.
  static const seedMaxValue = 9223372036854775807;
                              ^^^^^^^^^^^^^^^^^^^
/C:/Users/William/AppData/Local/Pub/Cache/hosted/pub.dev/openai_dart-0.0.2+1/lib/src/generated/schema/create_chat_completion_request.dart:154:31: Error: The integer literal 9223372036854775807 can't be represented exactly in JavaScript.

Try changing the literal to something that can be represented in JavaScript. In JavaScript 9223372036854775808 is the nearest value that can be represented exactly.
  static const seedMaxValue = 9223372036854775807;
                              ^^^^^^^^^^^^^^^^^^^
```

### Related Components

- [ ] doc-loaders
- [ ] doc-transformers
- [ ] prompts
- [X] llms
- [X] chat-models
- [ ] output-parsers
- [ ] chains
- [ ] memory
- [ ] stores
- [ ] embeddings
- [ ] retrievers
- [ ] agents

### Reproduction

Flutter web

Just this line: `import 'package:langchain_openai/langchain_openai.dart';`
And that line produces the same error with or without `import 'package:langchain/langchain.dart';` as well.

My pubspec.yaml:
```
dependencies:
  langchain: ^0.0.15
  langchain_openai: ^0.0.15+1
```

**Comment by davidmigloz** [comment]
Hi @bryantwilliam,

Thanks for reporting!

I've issued a fix in [`langchain_openai: 0.0.15+2`](https://pub.dev/packages/langchain_openai/changelog). I've run all the tests in the browser and all are green now. Let me know if it works fine on your side.

Cheers.

**Comment by bryantwilliam** [comment]
It works, thanks.
